pkg/webhook/validate_service_test.go

```diff
@@ -101,7 +101,7 @@ func TestServiceValidation(t *testing.T) {
 			got := ValidateRevisionTemplate(ctx, unstruct)
 			if got == nil {
 				if test.want != "" {
-					t.Errorf("Validate got='%v', want='%v'", got, test.want)
+					t.Errorf("Validate got=nil, want='%q'", test.want)
```

Comment:
```suggestion
					t.Errorf("Validate got=nil, want=%q", test.want)
```
`%q` already does quote escaping.